simpeg/electromagnetics/static/resistivity/sources.py

```diff
@@ -154,41 +159,46 @@ def __init__(
         location_a=None,
         location_b=None,
         location=None,
-        **kwargs,
+        current=(1.0, -1.0),
```

Comment:
I wouldn't support having different currents for each electrode on this class.

Reply:
That makes total sense! And the sign of the current for each electrode should be always `(+, -)`, right?

Reply:
If that's so, then I think we should only take a single float as the `current` argument for this method.